guides/doc-Administering_with_Ansible/master.adoc

```diff
@@ -1,11 +1,22 @@
+include::common/attributes.adoc[]
 :imagesdir: common/images
 :numbered:
 ifeval::["{build}" != "satellite"]
 :toc:
 :toc-placement: left
 endif::[]
-
-include::common/attributes.adoc[]
+ifeval::["{DocState}" == "nightly"]
+:revnumber: Nightly
+:revdate: published {date_my}
+endif::[]
+ifeval::["{DocState}" == "stable"]
+:revnumber: {ProjectVersion}
+:revdate: published {date_mdy}
+endif::[]
+ifeval::["{DocState}" == "unsupported"]
+:revnumber: {ProjectVersion} (unsupported)
+:revdate: published {date_mdy}
+endif::[]
```

Comment:
Is there a way to avoid this duplication everywhere? Perhaps via an include?

Reply:
How I could miss this! Of course.